### ISSUE TITLE
Enhancement - some way to access 'Add New World Info' button from anywhere in world info folders, rather than just at top

### ISSUE THREAD

**Issue by Feldherren** [ebolam/KoboldAI#356]
Currently you need to scroll back up to the top of a folder to add a new world info entry to it. This is fine when you have only one or two entries, but when you get to three or more you start to need to scroll to add more.

A floating button or just adding the 'add new world info' button to the bottom of the folder would help. Not sure about other solutions.

**Comment by valden80** [comment]
Alternative solution - to add new WI cards on top of list, shifting down older cards. So there will be minor needs to scroll up after every card.

**Comment by Feldherren** [comment]
Yeah. The exact order doesn't matter; that would work better than my suggestion.
